packages/flutter_tools/lib/src/test/flutter_platform.dart

```diff
@@ -0,0 +1,136 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:async/async.dart';
+import 'package:path/path.dart' as path;
+import 'package:stream_channel/stream_channel.dart';
+
+import 'package:test/src/backend/test_platform.dart';
+import 'package:test/src/runner/configuration.dart';
+import 'package:test/src/runner/plugin/platform.dart';
+import 'package:test/src/runner/loader.dart' as hack;
+
+final String _kSkyShell = Platform.environment['SKY_SHELL'];
+const String _kHost = '127.0.0.1';
+const String _kPath = '/runner';
+
+String shellPath;
+
+void installHook() {
+  hack.createVMPlatform = (Configuration config) => new FlutterPlatform(config);
```

Comment:
I'm curious, is this API more supported, or close to what a supported API will look like?